runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java

```diff
@@ -93,7 +93,7 @@ default boolean normalTeleportIcon()
 
 	@ConfigItem(
 		keyName = WorldMapPlugin.CONFIG_KEY_ANCIENT_TELEPORT_ICON,
-		name = "Show ancient magicks destionations",
+		name = "Show ancient magicks destinations",
```

Comment:
Ancient Magicks

Reply:
Now I'm conflicted on if Magicks should be capitalized.

Reply:
I would capitalise it. It's a title of a thing.

Reply:
Spellbook could also be capitalized too. I'm not sure how it is actually done in game or if it's even consistent in game. Basically that's why I originally just made it all lower case, it's easier :P

Reply:
Yeah, I can understand that, but I've long stopped looking at how Jagex does things for consistency.

Reply:
Wiki has them all as standard spellbook, Lunar spellbook, Arceuus spellbook and Ancient Magicks. I'm fine with capitalizing it all though.